services/policies/README.md

```diff
@@ -0,0 +1,70 @@
+# Policies Service
+
+The Policies Service provides a new grpc api which can be used to return whether a requested operation is allowed or not.
+Open Policy Agent is used to determine the set of rules of what is permitted and what is not.
+
+Policies are written in the [rego query language](https://www.openpolicyagent.org/docs/latest/policy-language/).
+
+The Policies Service consists of the following modules
+
+* Proxy Authorization (middleware)
+* Event Authorization (async post-processing)
+* GRPC API (can be used from other services too)
+
```

Comment:
```suggestion
The Policies Service provides a new grpc api which can be used to return whether a requested operation is allowed or not. To do so, Open Policy Agent (OPA) is used to determine the set of rules of what is permitted and what is not.

Policies are written in the [rego query language](https://www.openpolicyagent.org/docs/latest/policy-language/). The location of the rego files can be configured via yaml, a configuration via environment variables is not possible.

The Policies Service consists of the following modules:

*   Proxy Authorization (middleware)
*   Event Authorization (async post-processing)
*   GRPC API (can be used from other services too)

To configure the Policies Service, three environment variables need to be defined:

*   `POLICIES_ENGINE_TIMEOUT`
*   `POLICIES_POSTPROCESSING_QUERY`
*   `PROXY_POLICIES_QUERY`

Note that each query setting defines the [Complete Rules](https://www.openpolicyagent.org/docs/latest/#complete-rules) variable defined in the rego rule set the corresponding step uses for the evaluation. If the variable is mistyped or not found, the evaluation defaults to deny. Individual query definitions can be defined for each module.

To activate a the policies service for a module, it must be started with a yaml configuration that points to one or more rego files. Note that if the service is scaled horizontally, each instance should have access to the same rego files to avoid unpredictable results.
```